docs/features/wait/introduction.md

```diff
@@ -0,0 +1,22 @@
+# Wait Strategies
+
+!!! info "Wait strategies vs Startup strategies"
+
+    **Wait strategy:** is the container in a state that is useful for testing. This is generally approximated as 'can we talk to this container over the network'. However, there are quite a few variations and nuances.
+    
+    **Startup strategy:** did a container reach the desired running state. *Almost always* this just means 'wait until the container is running' - for a daemon process in a container this is the goal. Sometimes we need to wait until the container reaches a running state and then exits - this is the 'one shot startup' strategy, only used for cases where we need to run a one off command in a container but not a daemon.
```

Comment:
```suggestion
    **Startup strategy:** Did a container reach the desired running state? *Almost always* this just means 'wait until the container is running' - for a daemon process in a container this is the goal. Sometimes we need to wait until the container reaches a running state and then exits - this is the 'one shot startup' strategy, only used for cases where we need to run a one off command in a container but not a daemon.
```

Do we have `one shot startup` strategy in `tc-go`?

Reply:
You are right. Wrong copy&paste from Java docs. I've removed the one-shot strategy to use the `wait.ForExit`, which waits for the container to exit. What this strategy does is simply to verify if the container is not in the `Running` state, which we could improve to check the exit status of the container, like in Java

Reply:
A `StartupCheckStrategy` is an additional concept in `testcontainers-java`, that is verified before the configured `WaitStrategy` is checked. So if we don't yet have this additional layer in `tc-go`, I'd recommend removing it from the docs.